lib/chainweb/pact/spec.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Kadena.Chainweb.Pact.Spec do
```

Comment:
@santiagoboterokommit please, rename this module according to its context. IMO, `Spec` does not make sense and it is not a default name for contacts specification